### PR TITLE
add dependence; remove unused code

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

File: parl/remote/remote_decorator.py [hunk 1]
```diff
@@ -301,6 +301,22 @@ def __setattr__(self, attr, value):
         return proxy_wrapper
 
     max_memory = kwargs.get('max_memory')
-    if len(args) == 1 and callable(args[0]):
+    if len(args) == 1 and callable(args[0]):  # args[0]: cls
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/remote_decorator.py]: Could you explain why we need to check these conditions and add comment for it?

Comment (zenghsh3) [reply parl/remote/remote_decorator.py]: fixed

File: parl/remote/remote_decorator.py [hunk 2]
```diff
@@ -301,6 +301,14 @@ def __setattr__(self, attr, value):
         return proxy_wrapper
 
     max_memory = kwargs.get('max_memory')
-    if len(args) == 1 and callable(args[0]):
+    """
+        The `args` is different in the following two decorating way, and we should return different wrapper.
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/remote_decorator.py:308]: Users may pass some arguments to the decorator (e.g., parl.remote_class(10)). The following code tries to handle this issue.

Comment (zenghsh3) [reply parl/remote/remote_decorator.py:308]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**